### DISCUSSION TITLE
Deployment issue for default files.

### DISCUSSION THREAD

**Discussion by sheri0441** [vercel/next.js#63746] [Help] [answered]
### Summary

I have just started using Next.js. 

I am trying to deploy the default Nextjs project to Vercel for deployment purposes, but I keep getting this error.

""Your Team encountered an unknown problem. Please reach out to our support team for details.""

I have tried deploy the boilerplate Nextjs but still get this error.

Anyone, can help

**Comment by icyJoseph** [comment]
Hi,

I think for this kind of problem, you have to contact Vercel customer support. They can pinpoint exactly what has gone wrong in your deploy, as it looks like a customer issue, rather than a Next.js issue.

**Reply by sheri0441** [reply, marked as answer]
Thanks for your input. After some consideration, I find out the issue. I was using hobby plan and after changing the plan to pro trail the problem is solved.
